### PR TITLE
[FIX] website_slides: don't crash when the slide has been archived

### PR DESCRIPTION
**Pull request by rim-odoo**
opw-[2089868](https://www.odoo.com/web?debug#id=2089868&model=project.task&view_type=form&menu_id=5200)

### CODE REVIEW

**Comment by tde-banana-odoo** [comment]
What is the crash ? What is the issue ?

**Comment by rim-odoo** [comment]
This https://github.com/odoo/odoo/blob/saas-12.3/addons/website_slides/controllers/main.py#L171
fails when your slide has been unpublished but your partner was referenced by a slide.slide.partner

File: addons/website_slides/controllers/main.py [hunk 1]
```diff
@@ -165,7 +165,8 @@ def _get_channel_progress(self, channel, include_quiz=False):
         if not request.env.user._is_public() and channel.is_member:
             slide_partners = request.env['slide.slide.partner'].sudo().search([
                 ('channel_id', '=', channel.id),
-                ('partner_id', '=', request.env.user.partner_id.id)
+                ('partner_id', '=', request.env.user.partner_id.id),
+                ('slide_id.active', '=', True)
```
Comment (tde-banana-odoo) [inline addons/website_slides/controllers/main.py]: Should be slide_id, in, slides.ids to be coherent with above search

I noticed we have your fix in master but it does not seem correct to me.

Comment (rim-odoo) [reply addons/website_slides/controllers/main.py]: Hagrid

Comment (rim-odoo) [reply addons/website_slides/controllers/main.py]: @tde-banana-odoo _Not guilty, Your Honour!_ : https://github.com/odoo/odoo/commit/38b5a666c50d47a0efe060c031d68a2d70689dc6

(I did not notice this fix in master, master is not for tech support :p )

Comment (tde-banana-odoo) [reply addons/website_slides/controllers/main.py]: Well by your fix it was more "the same fix", not that you were the author, indeed ^^ .

Anyway slides.ids is better than checking active.

**Comment by tde-banana-odoo** [comment]
@robodoo r+

File: addons/website_slides/controllers/main.py (hunk 1 above)
Comment (rim-odoo) [reply addons/website_slides/controllers/main.py]: Done

**Comment by rim-odoo** [comment]
@tde-banana-odoo By the way, when fwbot wakes up, I'll override master fix with this one for consistency

**Pull request closed without merging**